<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabReportAutoBackupSettings</class>
 <widget class="QWidget" name="TabReportAutoBackupSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QWidget" name="widgetChannel" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelChannel">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBox" name="comboBoxType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelFileFormat">
       <property name="text">
        <string>File Format</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelBackupTimeRange">
       <property name="text">
        <string>Backup Time Range</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelBackupTo">
       <property name="text">
        <string>Backup to</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelReportAutoBackup">
       <property name="text">
        <string>Report Auto Backup</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="ComboBox" name="comboBoxFileFormat"/>
     </item>
     <item row="3" column="0">
      <widget class="QWidget" name="widgetGroup" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelGroup">
          <property name="text">
           <string>Group</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QWidget" name="widgetLengthOfStay" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ComboBox" name="comboBoxStay"/>
        </item>
        <item>
         <widget class="ComboBox" name="comboBoxStayTime"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="ComboBox" name="comboBoxDay"/>
     </item>
     <item row="6" column="1">
      <widget class="MyTimeEdit" name="timeEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelDay">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="CheckBoxGroup" name="checkBoxGroupChannel" native="true"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelTime">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="CheckBoxGroup" name="checkBoxGroupGroup" native="true"/>
     </item>
     <item row="9" column="1">
      <widget class="ComboBox" name="comboBoxTimeRange"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelLengthOfStay">
       <property name="text">
        <string>Length of Stay</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ComboBox" name="comboBoxAutoBackup"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelReportType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Report Type</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QWidget" name="widgetLine" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelLine">
          <property name="text">
           <string>Line</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="CheckBoxGroup" name="widgetLineType" native="true"/>
     </item>
     <item row="10" column="1">
      <widget class="QWidget" name="widgetBackupto" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxExtemalDevice">
          <property name="text">
           <string>Extemal Device</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxEmail">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>CheckBoxGroup</class>
   <extends>QWidget</extends>
   <header>checkboxgroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>mytimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
